src/axislike.rs

```diff
@@ -25,13 +25,14 @@ impl AxisDirection {
         }
     }
 
-    /// Checks if the given `value` represents an active input in this direction.
+    /// Checks if the given `value` represents an active input in this direction,
+    /// considering the specified `threshold`.
     #[must_use]
     #[inline]
-    pub fn is_active(&self, value: f32) -> bool {
+    pub fn is_active(&self, value: f32, threshold: f32) -> bool {
         match self {
-            Self::Negative => value < 0.0,
-            Self::Positive => value > 0.0,
+            Self::Negative => value < -threshold,
+            Self::Positive => value > threshold,
```

Comment:
I think I would use:
```
Self::Negative => value < -threshold.abs(),
Self::Positive => value > threshold.abs(),
```

This could prevent user pitfalls.